### ISSUE TITLE
How do I host this on a server?

### ISSUE THREAD

**Issue by adammertzenich** [Laverna/laverna#855]
How do I host the web version on my ubuntu server? Do I need to use the old webapp release or the static version of laverna?

**Comment by Garrettlynch** [comment]
Just installed it on my server - it had a dropbox bug until today so download it again.  Steps to installing below.

1. Download the static (again) from here: https://github.com/Laverna/static-laverna/archive/gh-pages.zip
2. Ftp it to your server.
3. If you need to password access to it go to Laverna on your server via your browser.  Go into Settings > Encryption.  Turn on Encryption and give it a password.  Click Save and Close.

That's it but if you are using dropbox to store your notes ignore step 3 for the moment.

4.  Follow the instructions here at the end of the page to set up a dropbox app: https://github.com/Laverna/laverna/wiki/FAQ.  Note the the redirect uri needs to be "https://www.yourserver.com/laverna/" (or whatever the url is, without quotes) the https is important so make sure it is enabled on your server.
5.  Once the app is made on dropbox go to Laverna on your server via your browser.  
6. Go into Settings > Sync.  Select Dropbox in the dropdown and copy and paste you're Dropbox API key from the Dropbox app page.  Click Save and Close.  If you did it right it will reload, attempt to authorise Dropbox and reload again.  Check by creating a note and then seeing if it appears in dropbox.
7.  Now do step 3 if you need to password it.

If you are using Dropbox don't create any notes of notebooks before you set it up otherwise they will save locally and you may get data loss and/or syncing issues once Dropbox is set up.

**Comment by e-volusian** [comment]
@Garrettlynch Are you actually seeing content populated in your Dropbox > App folder? On my end, everything seems to be working, but I see no evidence that anything is actually syncing.

**Comment by Garrettlynch** [comment]
Yes all working for me.  The error 400 I think is to do with the redirect uri on Dropbox.  Make sure it is correctly pointing to your installation of Laverna online - it needs to be something like "https://www.yourserver.com/laverna/".
